pandas/tests/groupby/__init__.py

```diff
@@ -0,0 +1,10 @@
+def get_method_args(name, obj):
```

Comment:
can you add a one-liner docstring here

Reply:
Thanks - good idea. I ended up doing a full docstring.